stock_multi_warehouse_security/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2019 Akretion
+# Copyright 2022 Foodles (http://www.foodles.co).
+# @author Pierre Verkest <pierreverkest84@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock Multi-Warehouse Security",
+    "version": "14.0.1.0.0",
+    "category": "Warehouse Management",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "author": "Akretion, Pierre Verkest, Odoo Community Association (OCA)",
+    "maintainers": ["petrus-v"],
+    "license": "AGPL-3",
+    "installable": True,
+    "summary": "Restrict user access in multi-warehouse environment",
+    "depends": [
+        "stock",
+        "sale_stock",
```

Comment:
Because I wanted to test, even user is not allowed to access to a given warehouse data, if employee is a saler he should be able to set a forbidden warehouse data access as default warehouse used while selling products cf https://github.com/OCA/stock-logistics-warehouse/pull/1543/files#diff-2fa3811d5404bb7e50aeb963eb796aeabafd818fb3f029d53c37468fbba3dae7R9. Probably I should only make sure read access on warehouse in my test and remove the dependency :thinking:

Reply:
I've remove the dependency an skip test if module is not installed